### PR TITLE
CID 115926 Dereference null return value

### PR DESCRIPTION
**Pull request by bazile-clyde**

### CODE REVIEW

File: src/mongocxx/test/client_side_encryption.cpp [hunk 1]
```diff
@@ -77,14 +77,12 @@ using namespace mongocxx;
 // Takes a path relative to the ENCRYPTION_TESTS_PATH variable, with leading '/'.
 bsoncxx::document::value _doc_from_file(stdx::string_view sub_path) {
     char* encryption_tests_path = std::getenv("ENCRYPTION_TESTS_PATH");
-    std::string path{encryption_tests_path};
-    if (path.back() == '/') {
-        path.pop_back();
```
Comment (bazile-clyde) [inline src/mongocxx/test/client_side_encryption.cpp:82]: This was never used.

File: examples/mongocxx/remove.cpp [hunk 2]
```diff
@@ -32,22 +32,14 @@ int main(int, char**) {
...
-        db["restaurants"].delete_many({});
-    }
+    { db["restaurants"].delete_many({}); }
 
     // Drop a collection.
-    {
-        db["restaurants"].drop();
-    }
+    { db["restaurants"].drop(); }
 }
```
Comment (bazile-clyde) [inline examples/mongocxx/remove.cpp:45]: clang_format

File: examples/mongocxx/index.cpp [hunk 3]
```diff
@@ -37,9 +37,7 @@ int main(int, char**) {
     }
 
     // Create a single field index.
-    {
-        db["restaurants"].create_index(make_document(kvp("cuisine", 1)), {});
-    }
+    { db["restaurants"].create_index(make_document(kvp("cuisine", 1)), {}); }
 
     // Create a compound index.
     {
```
Comment (bazile-clyde) [inline examples/mongocxx/index.cpp:43]: clang_format

File: examples/mongocxx/document_validation.cpp [hunk 4]
```diff
@@ -68,6 +68,5 @@ int main(int, char**) {
             std::cout << "Some zombie needs to eat more brains:" << std::endl;
             std::cout << e.what() << std::endl;
         }
-
```
Comment (bazile-clyde) [inline examples/mongocxx/document_validation.cpp:71]: clang_format

**Review by kevinAlbs — APPROVED** [review]
LGTM

**Pull request merged**